src/olympia/scanners/tests/test_admin.py

```diff
@@ -1764,25 +1763,3 @@ def test_matching_filenames_in_changelist(self):
             ),
         ]
         assert [link.attrib['href'] for link in links] == expected
-
-
-class TestIsSafeUrl(TestCase):
-    def test_enforces_https_when_request_is_secure(self):
```

Comment:
shouldn't we have these tests for amo/utils.py?

Reply:
I assumed `_is_safe_url `was already tested 😬 (I guess not)

Reply:
https://github.com/mozilla/addons-server/pull/17696/commits/dab2a4fed49b0adaee2b598c2a708798c16be34e